### PR TITLE
feat(plasma-ui): Prop `extraContent` for Confirm

### PR DESCRIPTION
**Pull request by kayman233**
## Release Notes

Добавлено свойство `extraContent`, принимающее `ReactNode`, который будет находится под основным контентом.

Снимок экрана 2023-08-16 в 15 29 10

-----

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.640.5876524963.ts'>borderRadius_default_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.640.5876524963.ts'>borderRadius_plasma_b2c_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.640.5876524963.ts'>borderRadius_plasma_web_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.640.5876524963.ts'>borderRadius_sberHealth_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.640.5876524963.ts'>borderRadius_sbermarket_business_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.640.5876524963.ts'>borderRadius_sbermarket_metro_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.640.5876524963.ts'>borderRadius_sbermarket_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.640.5876524963.ts'>borderRadius_sbermarket_selgros_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.640.5876524963.ts'>borderRadius_sberonline_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.640.5876524963.ts'>borderRadius_sberprime_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.640.5876524963.ts'>borderRadius_stylesSalute_react-native--canary.640.5876524963.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.640.5876524963.swift'>color_default_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.640.5876524963.kt'>color_default_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.640.5876524963.ts'>color_default_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.640.5876524963.xml'>color_default_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.640.5876524963.swift'>color_plasma_b2c_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.640.5876524963.kt'>color_plasma_b2c_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.640.5876524963.ts'>color_plasma_b2c_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.640.5876524963.xml'>color_plasma_b2c_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.640.5876524963.swift'>color_plasma_web_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.640.5876524963.kt'>color_plasma_web_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.640.5876524963.ts'>color_plasma_web_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.640.5876524963.xml'>color_plasma_web_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.640.5876524963.swift'>color_sberHealth_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.640.5876524963.kt'>color_sberHealth_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.640.5876524963.ts'>color_sberHealth_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.640.5876524963.xml'>color_sberHealth_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.640.5876524963.swift'>color_sbermarket_business_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.640.5876524963.kt'>color_sbermarket_business_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.640.5876524963.ts'>color_sbermarket_business_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.640.5876524963.xml'>color_sbermarket_business_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.640.5876524963.swift'>color_sbermarket_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.640.5876524963.kt'>color_sbermarket_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.640.5876524963.swift'>color_sbermarket_metro_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.640.5876524963.kt'>color_sbermarket_metro_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.640.5876524963.ts'>color_sbermarket_metro_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.640.5876524963.xml'>color_sbermarket_metro_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.640.5876524963.ts'>color_sbermarket_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.640.5876524963.swift'>color_sbermarket_selgros_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.640.5876524963.kt'>color_sbermarket_selgros_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.640.5876524963.ts'>color_sbermarket_selgros_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.640.5876524963.xml'>color_sbermarket_selgros_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.640.5876524963.swift'>color_sbermarket_wlbusiness_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.640.5876524963.kt'>color_sbermarket_wlbusiness_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts'>color_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.640.5876524963.xml'>color_sbermarket_wlbusiness_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.640.5876524963.xml'>color_sbermarket_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.640.5876524963.swift'>color_sberonline_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.640.5876524963.kt'>color_sberonline_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.640.5876524963.ts'>color_sberonline_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.640.5876524963.xml'>color_sberonline_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.640.5876524963.swift'>color_sberprime_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.640.5876524963.kt'>color_sberprime_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.640.5876524963.ts'>color_sberprime_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.640.5876524963.xml'>color_sberprime_xml--canary.640.5876524963.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.640.5876524963.swift'>color_stylesSalute_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.640.5876524963.kt'>color_stylesSalute_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.640.5876524963.ts'>color_stylesSalute_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.640.5876524963.xml'>color_stylesSalute_xml--canary.640.5876524963.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.640.5876524963.ts'>shadow_default_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.640.5876524963.ts'>shadow_plasma_b2c_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.640.5876524963.ts'>shadow_plasma_web_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.640.5876524963.ts'>shadow_sberHealth_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.640.5876524963.ts'>shadow_sbermarket_business_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.640.5876524963.ts'>shadow_sbermarket_metro_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.640.5876524963.ts'>shadow_sbermarket_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.640.5876524963.ts'>shadow_sbermarket_selgros_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts'>shadow_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.640.5876524963.ts'>shadow_sberonline_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.640.5876524963.ts'>shadow_sberprime_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.640.5876524963.ts'>shadow_stylesSalute_react-native--canary.640.5876524963.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.640.5876524963.ts'>spacing_default_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.640.5876524963.ts'>spacing_plasma_b2c_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.640.5876524963.ts'>spacing_plasma_web_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.640.5876524963.ts'>spacing_sberHealth_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.640.5876524963.ts'>spacing_sbermarket_business_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.640.5876524963.ts'>spacing_sbermarket_metro_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.640.5876524963.ts'>spacing_sbermarket_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.640.5876524963.ts'>spacing_sbermarket_selgros_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts'>spacing_sbermarket_wlbusiness_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.640.5876524963.ts'>spacing_sberonline_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.640.5876524963.ts'>spacing_sberprime_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.640.5876524963.ts'>spacing_stylesSalute_react-native--canary.640.5876524963.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.640.5876524963.swift'>typo_mage_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.640.5876524963.kt'>typo_mage_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.640.5876524963.ts'>typo_mage_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.640.5876524963.swift'>typo_plasma_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.640.5876524963.kt'>typo_plasma_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.640.5876524963.ts'>typo_plasma_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.640.5876524963.swift'>typo_ruler_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.640.5876524963.kt'>typo_ruler_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.640.5876524963.ts'>typo_ruler_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.640.5876524963.swift'>typo_sage_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.640.5876524963.kt'>typo_sage_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.640.5876524963.ts'>typo_sage_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.640.5876524963.swift'>typo_sbermarket_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.640.5876524963.kt'>typo_sbermarket_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.640.5876524963.ts'>typo_sbermarket_react-native--canary.640.5876524963.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.640.5876524963.swift'>typo_soulmate_ios-swift--canary.640.5876524963.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.640.5876524963.kt'>typo_soulmate_kotlin--canary.640.5876524963.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.640.5876524963.ts'>typo_soulmate_react-native--canary.640.5876524963.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.221.0-canary.640.5876524963.0
  npm install @salutejs/plasma-core@1.126.0-canary.640.5876524963.0
  npm install @salutejs/plasma-hope@1.221.0-canary.640.5876524963.0
  npm install @salutejs/plasma-icons@1.155.0-canary.640.5876524963.0
  npm install @salutejs/plasma-temple@1.170.0-canary.640.5876524963.0
  npm install @salutejs/plasma-tokens-b2b@1.25.0-canary.640.5876524963.0
  npm install @salutejs/plasma-tokens-b2c@0.34.0-canary.640.5876524963.0
  npm install @salutejs/plasma-tokens-web@1.40.0-canary.640.5876524963.0
  npm install @salutejs/plasma-tokens@1.59.0-canary.640.5876524963.0
  npm install @salutejs/plasma-ui@1.202.0-canary.640.5876524963.0
  npm install @salutejs/plasma-web@1.221.0-canary.640.5876524963.0
  npm install @salutejs/plasma-cy-utils@0.66.0-canary.640.5876524963.0
  npm install @salutejs/plasma-sb-utils@0.124.0-canary.640.5876524963.0
  npm install @salutejs/plasma-tokens-utils@0.30.0-canary.640.5876524963.0
  # or 
  yarn add @salutejs/plasma-b2c@1.221.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-core@1.126.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-hope@1.221.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-icons@1.155.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-temple@1.170.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-tokens-b2b@1.25.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-tokens-b2c@0.34.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-tokens-web@1.40.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-tokens@1.59.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-ui@1.202.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-web@1.221.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-cy-utils@0.66.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-sb-utils@0.124.0-canary.640.5876524963.0
  yarn add @salutejs/plasma-tokens-utils@0.30.0-canary.640.5876524963.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-640/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-640/
ui storybook: http://plasma.sberdevices.ru/pr/pr-640/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-640/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-640/
ui storybook: http://plasma.sberdevices.ru/pr/pr-640/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-640/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

File: website/plasma-ui-docs/docs/components/Confirm.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    );
+
+    return (
+        <>
+            <Button text="Show Confirm" onClick={() => setVisible(true)} />
+            <Confirm visible={visible} extraContent={extraContent} onApprove={hide} onDismiss={hide} view="primary" {...props} />
+        </>
+    );
+}
+```
```
Comment (neretin-trike) [inline website/plasma-ui-docs/docs/components/Confirm.mdx:88]: 👍

File: packages/plasma-ui/src/components/Confirm/Confirm.tsx [hunk 2]
```diff
@@ -112,17 +117,23 @@ const ConfirmRoot = styled.div<{ visible: boolean }>`
 
     background-color: ${backgroundPrimary};
 
+    transition: ${({ theme }) => (theme.lowPerformance ? 'unset' : 'transform 0.5s')};
+    ${({ visible }) =>
+        !visible && {
+            transform: 'translateY(-100%)',
+        }};
```
Comment (neretin-trike) [inline packages/plasma-ui/src/components/Confirm/Confirm.tsx:124]: а для чего тут понадобился отдельный транзишн?

File: packages/plasma-ui/src/components/Confirm/Confirm.component-test.tsx [hunk 3]
```diff
@@ -16,6 +18,13 @@ describe('Confirm', () => {
         dismissText,
     };
 
+    const extraContent = (
+        <div style={{ marginTop: '0.5rem', display: 'flex', alignItems: 'center' }}>
+            <IconHelp />
+            <Body3>Дополнительный контент</Body3>
+        </div>
+    );
```
Comment (neretin-trike) [inline packages/plasma-ui/src/components/Confirm/Confirm.component-test.tsx:26]: добавь пожалуйста ещё в сторибук этот пример

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-640/

File: packages/plasma-ui/src/components/Confirm/Confirm.tsx (hunk 2 above)
Comment (kayman233) [reply packages/plasma-ui/src/components/Confirm/Confirm.tsx:124]: а он у нас и был) Я его не трогал, просто диффалка так подсветила

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-640/
ui storybook: http://plasma.sberdevices.ru/pr/pr-640/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-640/temple-storybook/

File: packages/plasma-ui/src/components/Confirm/Confirm.component-test.tsx (hunk 3 above)
Comment (kayman233) [reply packages/plasma-ui/src/components/Confirm/Confirm.component-test.tsx:26]: добавил

File: packages/plasma-ui/src/components/Confirm/Confirm.tsx (hunk 2 above)
Comment (neretin-trike) [reply packages/plasma-ui/src/components/Confirm/Confirm.tsx:124]: страненько 🤔

**Review by neretin-trike — APPROVED** [review]

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-640/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-640/
ui storybook: http://plasma.sberdevices.ru/pr/pr-640/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-640/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Review by neretin-trike — APPROVED** [review]

File: packages/plasma-ui/src/components/Confirm/Confirm.tsx [hunk 4]
```diff
@@ -213,9 +229,14 @@ export const Confirm = (props: ConfirmProps) => {
...
-                <StyledCell content={<TextBox title={title} subTitle={subtitle} />} />
-                {buttons}
+            <ConfirmRoot visible>
+                <ConfirmContainer>
+                    <ConfirmMain>
+                        <StyledCell content={<TextBox title={title} subTitle={subtitle} />} />
+                        {buttons}
+                    </ConfirmMain>
+                </ConfirmContainer>
+                <ConfirmFooter>{extraContent}</ConfirmFooter>
```
Comment (Yeti-or) [inline packages/plasma-ui/src/components/Confirm/Confirm.tsx:239]: мы не хотим чтобы у ConfirmFooter были такие же отступы слева и справа как у ConfirmContainer ??

```
margin: 1 rem

```
почему не хотим докинуть?

Comment (Yakutoc) [reply packages/plasma-ui/src/components/Confirm/Confirm.tsx:239]: @Yeti-or @neretin-trike  
Из нашего общего обсуждения я так понял, что все отступы внешние/внутренние отданы на откуп пользователю этого компонента.

Comment (Yakutoc) [reply packages/plasma-ui/src/components/Confirm/Confirm.tsx:241]: Обсудили что отступы R\L для `ConfirmFooter` задаем мы.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-640/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-640/
ui storybook: http://plasma.sberdevices.ru/pr/pr-640/ui-storybook/
temple storybook: http://plasma.sberdevices.ru/pr/pr-640/temple-storybook/

**Pull request merged**